lib/flutter_camera_ml_vision.dart

```diff
@@ -216,6 +215,7 @@ class CameraMlVisionState<T> extends State<CameraMlVision<T>> {
 
   @override
   void dispose() {
+    if (widget.onDispose != null) widget.onDispose();
```

Comment:
not sure what the analyzer say about this but we generally put bracket even for one line generally

Reply:
As resolved in https://github.com/rushio-consulting/flutter_camera_ml_vision/pull/67/commits/a3e924a22edea44b07a84ee1005b0aa07bc4fa49